doctor_visits/delphi_doctor_visits/sensor.py

```diff
@@ -168,7 +169,8 @@ def fit(y_data,
         Args:
             y_data: dataframe for one geo_id, with all 7 cols
             fit_dates: list of sorted datetime for which to use as training
-            sensor_dates: list of sorted datetime for which to produce sensor values
+            burn_in_dates: list of sorted datetime for which to produce sensor values
```

Comment:
why the rename to `burn_in` here? perhaps the docstring should be updated since; i associate burn in with values that are used as buffer before producing actual values

Reply:
The rename reflects that we are calling this with `burn_in_dates` rather than `sensor_dates` in the caller function, but I don't totally understand why the earlier naming convention was used.  I think there is some unnecessary complexity here that is reflected in the numerous reindex operations throughout.  At a high level, it is all around performing the fit over a larger window than we actually want to write output for and accounting for some lags in the data.  I was having trouble disentangling how to migrate indexing, etc. to a single spot so I just submitted.

Reply:
got it :+1: